05-visualisation-ggplot2.Rmd

```diff
@@ -152,7 +152,7 @@ ggplot(surveys_complete)
 - define aestetics (`aes`), that maps variables in the data to axes on the plot
      or to plotting size, shape color, etc.,
 
-```{r}
+```{r, eval=FALSE}
```

Comment:
You can also set `error=TRUE` as a chunk option to have it executed, but not stop and instead display the error message, if you want to demonstrate that and how something fails.